### PR TITLE
[Workers] services pictures and instructions

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/workers/src/content/learning/using-services/index.md [hunk 1]
```diff
@@ -5,31 +5,39 @@ pcx-content-type: concept
...
 
-Service environments take a cleaner approach. You can create and edit environments directly in the dashboard. Log in to the Cloudflare dashboard -> select your **Account** > **Workers** > select your **Service** > **Quick edit**.
+Workers Service environments take a cleaner approach. You can create and edit environments directly in the dashboard. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) > select your **Account** > **Workers** > select your **Workers Service** > **Quick edit**.
+
+Unlike Wrangler environments, Workers Service environments do not create extra scripts. They are, however, able to connect to their own KV stores and Durable Objects. The code for any environment can be changed directly in the dashboard via the quick editor, by going to **Account Home** > **Workers** > **choose your Worker** > **Quick edit**.
```
Comment (deadlypants1973) [inline products/workers/src/content/learning/using-services/index.md]: @eidam

File: products/workers/src/content/learning/using-services/index.md [hunk 2]
```diff
@@ -49,7 +57,7 @@ Workers Service bindings are in closed beta as of January 2022. Go to the [Worke
 
 ![service bindings settings](./media/service-bindings.png)
 
-A Workers Service binding allows you to send HTTP requests to another Worker without those HTTP requests going over the Internet. Because the request doesn't leave the Cloudflare edge network, latency is reduced and performance improves. You can invoke other Workers directly from your code because Service bindings allow more composability. In the example below, requests are validated by an authentication service.
+A Workers Service binding allows you to send HTTP requests to another Worker without those HTTP requests going over the Internet. Because the request does not leave the Cloudflare edge network, latency is reduced and performance improves. You can invoke other Workers directly from your code because Service bindings allow more composability. In the example below, requests are validated by an authentication Workers Service.
```
Comment (deadlypants1973) [inline products/workers/src/content/learning/using-services/index.md]: @eidam

File: products/workers/src/content/learning/using-services/index.md [hunk 3]
```diff
@@ -5,31 +5,39 @@ pcx-content-type: concept
...
 ## Service environments
 
 <Aside type="note">
 
-As of January 2022, the dashboard is the only way to interact with Service environments. Support in Wrangler is coming in [v2.1](https://github.com/cloudflare/wrangler2/issues/27)
+As of January 2022, the dashboard is the only way to interact with Workers Service environments. Support in Wrangler is coming in [v2.1](https://github.com/cloudflare/wrangler2/issues/27)
 
 </Aside>
 
 Wrangler supports an older version of environments. With Wrangler environments, you create custom contexts for your code to run in by adding keys to your `wrangler.toml` file. Wrangler will then generate a separate script for each environment. If you make a staging and production environment, for example, Wrangler will generate `my-worker-staging` and `my-worker-prod`.
```
Comment (eidam) [inline products/workers/src/content/learning/using-services/index.md]:
```suggestion
Wrangler supports an older version of environments. With Wrangler environments, you create custom contexts for your code to run in by adding keys to your `wrangler.toml` file. Wrangler will then generate a separate Workers Service for each Wrangler environment. If you make a staging and production environment, for example, Wrangler will generate `my-worker-staging` and `my-worker-prod`. 
```

File: products/workers/src/content/learning/using-services/index.md (hunk 1 above)
Comment (eidam) [inline products/workers/src/content/learning/using-services/index.md]: scripts are gone, replaced scripts with Workers Services
```suggestion
Unlike Wrangler environments, Workers Service environments do not create extra Workers Services. They are, however, able to connect to their own KV stores and Durable Objects. The code for any environment can be changed directly in the dashboard via the quick editor, by going to **Account Home** > **Workers** > **choose your Worker** > **Quick edit**.
```

File: products/workers/src/content/learning/using-services/index.md [hunk 4]
```diff
@@ -125,11 +132,11 @@ export default {
 }
 ```
 
-This authentication service does not need to have `*.workers.dev` or custom domain public endpoint, it is accessed through a Workers Service binding from the other Workers Service scripts directly.
+This authentication Workers Service does not need to have `*.workers.dev` or custom domain public endpoint, it is accessed through a Workers Service binding from the other Workers Service scripts directly.
```
Comment (eidam) [inline products/workers/src/content/learning/using-services/index.md]:
```suggestion
This authentication Workers Service does not need to have `*.workers.dev` or custom domain public endpoint, it is accessed through a Workers Service binding from the other Workers Service environment script directly.
```

File: products/workers/src/content/learning/using-services/index.md [hunk 5]
```diff
@@ -5,31 +5,45 @@ pcx-content-type: concept
 
 # Workers Services
 
-Services are the new building block for deploying applications on Cloudflare Workers. Workers Services are made of environments, which are scripts that can contain bindings to KV stores, Durable Objects, or even other services, as well as environment variables and secrets. Workers Services can have multiple environments and can set up pipelines for promoting a service from one environment to another.
+Workers Services are the new building blocks for deploying applications on Cloudflare Workers. Workers Services are made of environments, which are scripts that can contain bindings to KV stores, Durable Objects, or even other services, as well as environment variables and secrets. Workers Services can have multiple environments and can set up pipelines for promoting a Service from one environment to another.
```
Comment (kabirsikand) [inline products/workers/src/content/learning/using-services/index.md]:
```suggestion
Workers Services are the new building blocks for deploying applications on Cloudflare Workers. Workers Services are made of environments, which are scripts that can contain bindings to KV stores, Durable Objects, or even other services, as well as environment variables and secrets. Workers Services can have multiple environments and can set up pipelines for promoting a deployment from one environment to another.
```

File: products/workers/src/content/learning/using-services/index.md [hunk 6]
```diff
@@ -5,31 +5,45 @@ pcx-content-type: concept
...
 
-Unlike a script, a Workers Service is composable, which allows services to talk to each other; allowing you to develop new kinds of services like routers, middlewares, or traffic gateways. Services also support multiple environments, allowing you to test changes in a preview environment, then promote those changes to production when you are ready.
+<Aside type="note" header="Workers versus Workers Services?">
+
+Workers Services are synonymous with Workers. In the legacy Workers legacy model, Workers were created by writing scripts and Workers scripts could not communicate with each other. In the Workers Services model, Workers Services are made up of environments that are created by Worker scripts. The introduction of Workers Services adds Worker-to-Worker communication functionality.
```
Comment (kabirsikand) [inline products/workers/src/content/learning/using-services/index.md]:
```suggestion
Workers Services are synonymous with Workers. In the legacy Workers legacy model, Workers were created by writing scripts. Workers scripts could not communicate with each other. In the Workers Services model, Workers Services are made up of Workers Environments that are a collection of bindings, environment variables, secrets, and script deployments. The introduction of Workers Services also adds Worker-to-Worker communication functionality via Workers Service bindings.
```

File: products/workers/src/content/learning/using-services/index.md [hunk 7]
```diff
@@ -5,31 +5,45 @@ pcx-content-type: concept
...
+
+</Aside>
+
+Unlike a traditional Workers script, a Workers Service is composable, which allows Workers services to talk to each other; allowing you to develop new kinds of services like routers, middlewares, or traffic gateways. Workers Services also support multiple environments, allowing you to test changes in a preview environment, then promote those changes to production when you are ready.
```
Comment (kabirsikand) [inline products/workers/src/content/learning/using-services/index.md]:
```suggestion
Unlike a traditional Workers script, a Workers Service is composable, which allows Workers services to talk to each other; allowing you to develop new kinds of services like routers, middlewares, deployment managers, or traffic gateways. Workers Services also support multiple environments, allowing you to test changes in a preview environment, then promote those changes to production when you are ready – or execute your own custom deployment strategies.
```

File: products/workers/src/content/learning/using-services/index.md [hunk 8]
```diff
@@ -49,7 +63,7 @@ Workers Service bindings are in closed beta as of January 2022. Go to the [Worke
 
 ![service bindings settings](./media/service-bindings.png)
 
-A Workers Service binding allows you to send HTTP requests to another Worker without those HTTP requests going over the Internet. Because the request doesn't leave the Cloudflare edge network, latency is reduced and performance improves. You can invoke other Workers directly from your code because Service bindings allow more composability. In the example below, requests are validated by an authentication service.
+A Workers Service binding allows you to send HTTP requests to another Worker without those HTTP requests going over the Internet. Because the request does not leave the Cloudflare edge network, latency is reduced and performance improves. You can invoke other Workers directly from your code because Service bindings allow more composability. In the example below, requests are validated by an authentication Workers Service.
```
Comment (kabirsikand) [inline products/workers/src/content/learning/using-services/index.md]:
```suggestion
A Workers Service binding allows you to send HTTP requests to another Worker without those requests going over the Internet. The request immediately invokes the downstream Worker, reducing latency as compared to a request to a third-party service. You can invoke other Workers directly from your code; Workers Service bindings allow for much more composability on the Workers platform. In the example below, requests are validated by an authentication Workers Service.
```

File: products/workers/src/content/learning/using-services/index.md (hunk 4 above)
Comment (kabirsikand) [reply products/workers/src/content/learning/using-services/index.md]:
```suggestion
This authentication Workers Service does not need to have a `*.workers.dev` or other domain endpoint, nor does it need an HTTP Route: it is accessed through a Workers Service binding from the other Worker directly. The authentication Worker is, effectively, a private Worker Service.
```

**Review by eidam — APPROVED** [review]

File: products/workers/src/content/learning/using-services/index.md [hunk 9]
```diff
@@ -5,31 +5,45 @@ pcx-content-type: concept
...
 
-Unlike a script, a Workers Service is composable, which allows services to talk to each other; allowing you to develop new kinds of services like routers, middlewares, or traffic gateways. Services also support multiple environments, allowing you to test changes in a preview environment, then promote those changes to production when you are ready.
+<Aside type="note" header="Workers versus Workers Services?">
+
+Workers Services are synonymous with Workers. In the legacy Workers legacy model, Workers were created by writing scripts. Workers scripts could not communicate with each other. In the Workers Services model, Workers Services are made up of Workers Environments that are a collection of bindings, environment variables, secrets, and script deployments. The introduction of Workers Services also adds Worker-to-Worker communication functionality via Workers Service bindings.
```
Comment (kodster28) [inline products/workers/src/content/learning/using-services/index.md]:
```suggestion
Workers Services are synonymous with Workers. In the legacy Workers model, Workers were created by writing scripts. Workers scripts could not communicate with each other. In the Workers Services model, Workers Services are made up of Workers Environments that are a collection of bindings, environment variables, secrets, and script deployments. The introduction of Workers Services also adds Worker-to-Worker communication functionality via Workers Service bindings.
```

File: products/workers/src/content/learning/using-services/index.md [hunk 10]
```diff
@@ -5,31 +5,45 @@ pcx-content-type: concept
...
 
 </Aside>
 
-Wrangler supports an older version of environments. With Wrangler environments, you create custom contexts for your code to run in by adding keys to your `wrangler.toml` file. Wrangler will then generate a separate script for each environment. If you make a staging and production environment, for example, Wrangler will generate `my-worker-staging` and `my-worker-prod`. 
+Wrangler supports an older version of environments. With Wrangler environments, you create custom contexts for your code to run in by adding keys to your `wrangler.toml` file. Wrangler will then generate a separate Workers Service for each Wrangler environment. If you make a staging and production environment, for example, Wrangler will generate `my-worker-staging` and `my-worker-prod`. 
+
+Workers Service environments take a cleaner approach. You can create and edit environments directly in the dashboard. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) > select your **Account** > **Workers** > select your **Workers Service** > **Quick edit**.
```
Comment (kodster28) [inline products/workers/src/content/learning/using-services/index.md]: Maybe break this into numbered steps?

File: products/workers/src/content/learning/using-services/index.md [hunk 11]
```diff
@@ -5,31 +5,45 @@ pcx-content-type: concept
...
+
+Unlike Wrangler environments, Workers Service environments do not create extra Workers Services. They are, however, able to connect to their own KV stores and Durable Objects. The code for any environment can be changed directly in the dashboard via the quick editor, by going to **Account Home** > **Workers** > **choose your Worker** > **Quick edit**.
```
Comment (kodster28) [inline products/workers/src/content/learning/using-services/index.md]: Same comment here.

**Review by kodster28 — APPROVED** [review]

File: products/workers/src/content/learning/using-services/index.md (hunk 10 above)
Comment (deadlypants1973) [inline products/workers/src/content/learning/using-services/index.md]:
```suggestion
Workers Service environments take a cleaner approach. You can create and edit environments directly in the dashboard:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/).
2. Select your **Account**.
3. Go to **Workers**. 
4. Select your **Workers Service**.
5. Select  **Quick edit**.
```

File: products/workers/src/content/learning/using-services/index.md (hunk 11 above)
Comment (deadlypants1973) [inline products/workers/src/content/learning/using-services/index.md]:
```suggestion
Unlike Wrangler environments, Workers Service environments do not create extra Workers Services. They are, however, able to connect to their own KV stores and Durable Objects. The code for any environment can be changed directly in the dashboard via the quick editor:

1.  Go to **Account Home**.
2. Go to **Workers**. 
3. hoose your **Worker**.
4. Select **Quick edit**.
```

**Pull request merged**